packages/rules-unit-testing/package.json

```diff
@@ -16,7 +16,8 @@
     "dev": "rollup -c -w",
     "test:nyc": "TS_NODE_CACHE=NO TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --reporter lcovonly -- mocha 'test/{,!(browser)/**/}*.test.ts' --config ./mocharc.node.js",
     "test": "(cd functions && yarn) && firebase --project=demo-foo --debug emulators:exec 'yarn test:nyc'",
-    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test"
+    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test",
+    "api-report": "api-extractor run --local --verbose"
```

Comment:
Can you add a command to generate docs locally, like other packages?

Reply:
Added.